lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -56,7 +56,7 @@ class CategoryRenderer {
         title: Util.UIStrings.passedAuditsGroupTitle,
         className: 'lh-clump--passed',
       },
-      'not-applicable': {
+      'notApplicable': {
         title: Util.UIStrings.notApplicableAuditsGroupTitle,
         className: 'lh-clump--not-applicable',
```

Comment:
need to update

Reply:
oh we should get a puppeteer screenshot test in for the report to catch these css mismatches

Reply:
> oh we should get a puppeteer screenshot test in for the report to catch these css mismatches

good idea. Don't we also have a jsdom test or two looking for the not applicable results? Concerning they weren't found...

Reply:
> Concerning they weren't found...

oh, because the test selectors weren't updated either :)

But now I'm questioning if it should be updated :) We don't currently create the clump names dynamically, so it can be whatever we want. Commonly `notApplicable` in js goes to `not-applicable` in css, but do we care? Or is `notapplicable` fine? Keeping camelCase seems gross, though.

Reply:
I would go for not-applicable makes it easier to read in my opinion